source/_drafts/Autoplay-Best-Practices-with-Video-js.md

```diff
@@ -0,0 +1,113 @@
+---
+title: "Autoplay Best Practices with Video.js"
+author:
+  name: "Pat O'Neill"
+  github: misteroneill
+tags:
+---
+
+There has been a lot of chatter about autoplay lately and we wanted to take a few minutes to share some best practices when using autoplay with Video.js.
+
+This post is meant to be a source of information - not an editorial or positional statement on autoplaying video. A lot of users hate autoplay. And a lot of publishers providing content free of charge rely on autoplay for the preroll ads that finance their businesses. Browser vendors (and open source library authors) have to weigh these concerns based on the interests of their users, publishers across the web, and their own business. And users have to choose the browser that best reflects their preferences.
+
+This post is not about ways to circumvent autoplay policies. That's not possible and no one should attempt it. One positional statement everyone working on Video.js should be comfortable making is: actively circumventing the browser's built-in user experience or user preferences is harmful.
+
+## Autoplay in the Big Browsers
+I'm not going to go into exhaustive detail on each browser's idiosyncrasies because that defeats the point: **never assume autoplay will work.**
+
+### Chrome
+Back in September 2017, Google announced that [Chrome's autoplay policy would change in April 2018 with Chrome 66](https://developers.google.com/web/updates/2017/09/autoplay-policy-changes). In short, muted autoplay is allowed, but autoplay with sound will be subject to a series of rules depending on platform and previous video engagement (referred to as the "Media Engagement Index") with the specific domain.
```

Comment:
should we mention the gesture thing?

Reply:
I was intentionally trying not to explain each browser's particular algorithm because that's more likely to change over time and people can read about it from the browser vendors themselves, if they care.

Reply:
sounds good.